src/index.html

```diff
@@ -2,16 +2,193 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="inputForm"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldsetBlock">
+        <legend>Personal information</legend>
+        <div class="inputBlock">
+          <label for="surname">Surname:
+            <input
+              autocomplete="off"
+              id="surname"
+              type="text"
+              name="surname"
+            >
+          </label>
+        </div>
+
+        <div class="inputBlock">
+          <label for="name">Name:
+            <input
+              autocomplete="off"
+              id="name"
+              type="text"
+              name="name"
+            >
+          </label>
+        </div>
+
+        <div class="inputBlock">
+          <label for="age">How old are You?
+            <input
+              id="age"
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+
+        </div>
```

Comment:
you don't need blank lines between parent and child tags
```suggestion
          </label>
        </div>
```